examples/simple/transform.py

```diff
@@ -6,16 +6,16 @@
 @click.option('--input-path', required=True)
 @click.option('--output-path', required=True)
 @click.option('--multiplier', default=1)
-@click.option('--sqrt/--no-sqrt', is_flag=True, default=False)
+@click.option('--sqrt', default='False')
```

Comment:
this should just be a bool

Reply:
you were right that the flag needed to be changed, but if you set `default=True` this would probably be the best way to fix